src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAjaxPageData.java

```diff
@@ -21,4 +22,39 @@
     public InstructorFeedbackResponseCommentAjaxPageData(AccountAttributes account) {
         super(account);
     }
+
+    public FeedbackResponseComment getComment() {
+        FeedbackResponseComment frc = new FeedbackResponseComment(comment, comment.giverEmail);
+
+        frc.setResponseVisibleToRecipient(true);
+        frc.setResponseVisibleToGiverTeam(true);
+        frc.setResponseVisibleToRecipientTeam(true);
+        frc.setResponseVisibleToStudents(true);
+        frc.setResponseVisibleToInstructors(true);
+        frc.setEditDeleteEnabled(true);
+        frc.setInstructorAllowedToDelete(true);
+        frc.setInstructorAllowedToEdit(true);
+        frc.setResponseGiverName(giverName);
+        frc.setResponseRecipientName(recipientName);
+        frc.setShowCommentToString(showCommentToString);
+        frc.setShowGiverNameToString(showGiverNameToString);
+
+        return frc;
+    }
+
+    public String getCommentIdFirstIndex() {
+        return commentId.split("-")[0];
+    }
+
+    public String getCommentIdSecondIndex() {
+        return commentId.split("-")[1];
+    }
+
+    public String getCommentIdThirdIndex() {
+        return commentId.split("-")[2];
+    }
+
+    public String getCommentIdFrcIndex() {
+        return commentId.split("-")[3];
```

Comment:
This can be placed into a list and accessed by index on the frontend.
Alternatively, you can find out exactly what each of these indexes are meant for and give the methods proper names.
The data can be stored on the backend as separate Strings or a list, but it may be good to break it up beforehand, rather than on call.
(i.e. `{ commentIds = commentId.split("-"); } . . . { return commentsIds[*] }`)